data/mods/Magiclysm/enchantments/Sun_Mage.json

```diff
@@ -14,7 +14,8 @@
     "min_range": 1,
     "max_range": 1,
     "min_duration": 200,
-    "max_duration": 200
+    "max_duration": 200,
+    "damage_type": "pure"
```

Comment:
```suggestion
    "damage_type": "pure"
```
it's really late for me here so i can't double check, but this should be either heat or fire, whichever is the right json value.